src/components/shared/form_fields/TextField.vue

```diff
@@ -22,7 +22,7 @@
 			@blur="$emit('field-changed', name )"
 			@update:modelValue="onUpdateModel"
 		/>
-		<span v-if="showError" class="help is-danger" :id="`${inputId}-error`">{{ errorMessage }}</span>
+		<span v-if="showError && errorMessage !== ''" class="help is-danger" :id="`${inputId}-error`">{{ errorMessage }}</span>
```

Comment:
Nitpick: I'd rather have the parent class not pass an empty error message than having to check for it. 
Alternatively, deprecate `showError` or make it a computed property. I don't remember what we were thinking when we decided to have two props that are kind of dependent on each other. In hindsight, it doesn't make sense to me and was probably driven by a misguided urge to be "explicit".

Reply:
Until now we passed a basically static string as the error message to the `TextField` componant and then toggled `showError` for the display, so having 2 props makes sense.

However in the calculator we have a generic error message rather than an error on each field, but we still need to mark the fields as invalid which makes an empty `help is-danger` element appear on the page and the padding/margin on it causes a gap. So that double check is `I should show an error message and I have an error message`

Reply:
Actually I should check if the aria-describedby is working correct on those fields too

Reply:
So good catch, the `aria-describedby` wasn't being set properly on the calculator fields. I've added a new optional boolean prop to tell the `TextField` that a custom error message is being used, and removed the empty string checks.